pycytominer/cyto_utils/load.py

```diff
@@ -1,8 +1,48 @@
 import csv
 import gzip
+from pathlib import Path
 import numpy as np
 import pandas as pd
 
+def is_path_a_parquet_file(file: str) -> bool:
+    """Checks if the provided file path is a parquet file.
+
+    Identification of parquet files are done by inspecting the file extensions. 
```

Comment:
```suggestion
    Identify parquet files by inspecting the file extensions. 
```

to avoid passive voice